app/models/user.rb

```diff
@@ -6,7 +6,10 @@ class User
          :recoverable, :rememberable, :trackable, :validatable
 
   # Non-devise generated
-  field :name, type: String
+  field :first_name, type: String
+  field :last_name, type: String
+  field :line, type: String
```

Comment:
re: line as a user attribute, I want to flag that this raises a UX question of how we want to handle DCAF being split into multiple geographies; does this attribute assign a CM to a line, or mark the last line they were on, or something else? 

But that's down the road a ways, so I think this is good for the time being, and mongo's flexible enough that we can change it down the road.